### PR TITLE
rdfind: add page

### PR DESCRIPTION
**Pull request by derphilipp**
The tool is a available from https://rdfind.pauldreik.se/

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/rdfind.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# rdfind
+
+> Finds duplicate files and gets rid of them.
+
+- Identify all duplicates in a given directory and output summary:
```
Comment (sbrl) [inline pages/common/rdfind.md]: Perhaps ...`output a summary` would read better?

File: pages/common/rdfind.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Replace all duplicates with hardlinks:
+
+`rdfind -makehardlinks true {{path/to/directory}}`
+
+- Replace all duplicates with symlinks/soft links:
+
+`rdfind -makesymlinks true {{path/to/directory}}`
+
+- Delete all duplicates, do not ignore empty files:
```
Comment (sbrl) [inline pages/common/rdfind.md]: Perhaps ...`duplicates, and do not ignore`.... would read better?

File: pages/common/rdfind.md (hunk 1 above)
Comment (derphilipp) [reply pages/common/rdfind.md]: Good point! 👍

File: pages/common/rdfind.md (hunk 2 above)
Comment (derphilipp) [reply pages/common/rdfind.md]: Valid point, i like that one better; Thank you

File: pages/common/rdfind.md [hunk 3]
```diff
@@ -14,6 +14,6 @@
 
 `rdfind -makesymlinks true {{path/to/directory}}`
 
-- Delete all duplicates, do not ignore empty files:
+- Delete all duplicates, and do not ignore empty files:
```
Comment (mfrw) [inline pages/common/rdfind.md]: Drop the `,`

Comment (derphilipp) [reply pages/common/rdfind.md]: Done 😄

File: pages/common/rdfind.md (hunk 2 above)
Comment (mfrw) [reply pages/common/rdfind.md]: I think the `,` is superfluous. We should probably drop it.

Comment (derphilipp) [reply pages/common/rdfind.md]: Agreed, removed it;

File: pages/common/rdfind.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+`rdfind -makehardlinks true {{path/to/directory}}`
+
+- Replace all duplicates with symlinks/soft links:
+
+`rdfind -makesymlinks true {{path/to/directory}}`
+
+- Delete all duplicates and do not ignore empty files:
+
+`rdfind -deleteduplicates true -ignoreempty false {{/mnt/backup}}`
```
Comment (agnivade) [inline pages/common/rdfind.md]: Why is this suddenly `/mnt/backup` ?

File: pages/common/rdfind.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# rdfind
+
+> Finds duplicate files and gets rid of them.
```
Comment (agnivade) [inline pages/common/rdfind.md]: I think its clearer to mention - "Finds files with duplicate names". Duplicate files might mean files with the same content too.

Comment (derphilipp) [reply pages/common/rdfind.md]: But it does find files of the same content, not the same name; I try to clarify that

Comment (agnivade) [reply pages/common/rdfind.md]: Ah, I understood the opposite. More the reason we should fix this. I suggest - `Find files with duplicate content ..`

Comment (derphilipp) [reply pages/common/rdfind.md]: Added a pull request to resolve the issue:  https://github.com/tldr-pages/tldr/pull/2341

File: pages/common/rdfind.md (hunk 4 above)
Comment (sbrl) [reply pages/common/rdfind.md]: Ooh good spot. I guess `path/to/backup` would have been better?

Comment (derphilipp) [reply pages/common/rdfind.md]: or just set it to a generic pathname like the rest;

**Pull request merged**